src/test/java/com/amazon/ion/impl/IonReaderContinuableTopLevelBinaryTest.java

```diff
@@ -4323,4 +4360,80 @@ public void readClobValueViaGetBytes(@ConvertWith(TestUtils.HexStringToByteArray
+        String inputBytes
+    ) throws Exception {
+        reader = readerForIon11(hexStringToByteArray(inputBytes), constructFromBytes);
+        assertSequence(
+            next(IonType.INT), expectation.apply(new String[] {"name"}), intValue(0),
+            next(IonType.INT), expectation.apply(new String[] {"symbols", "name"}), intValue(0),
+            next(IonType.INT), expectation.apply(new String[] {"name", "symbols", "imports"}), intValue(0),
+            next(null)
+        );
+        closeAndCount();
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "E4 09 50 E5 0F 09 50 E6 07 09 0F 0D 50", // SIDs
+        "E7 F9 6E 61 6D 65 50 E8 0F F9 6E 61 6D 65 50 E9 1D F9 6E 61 6D 65 0F F3 69 6D 70 6F 72 74 73 50", // FlexSyms
+        "E4 12 00 50 E5 0F 24 00 00 50 E6 0E 00 09 0F 0D 50", // SIDs (multi-byte FlexUInts)
+        "E7 F2 FF 6E 61 6D 65 50 E8 3C 00 00 F9 6E 61 6D 65 50 E9 F8 00 00 00 F9 6E 61 6D 65 0F E6 FF 69 6D 70 6F 72 74 73 50", // Multi-byte FlexSyms
+    })
+    public void readAnnotations_1_1(String inputBytes) throws Exception {
+        assertAnnotationsCorrectlyParsed(true, IonReaderContinuableTopLevelBinaryTest::annotations, inputBytes);
+        assertAnnotationsCorrectlyParsed(true, IonReaderContinuableTopLevelBinaryTest::annotationSymbols, inputBytes);
+        assertAnnotationsCorrectlyParsed(true, IonReaderContinuableTopLevelBinaryTest::annotationsIterator, inputBytes);
+        assertAnnotationsCorrectlyParsed(false, IonReaderContinuableTopLevelBinaryTest::annotations, inputBytes);
+        assertAnnotationsCorrectlyParsed(false, IonReaderContinuableTopLevelBinaryTest::annotationSymbols, inputBytes);
+        assertAnnotationsCorrectlyParsed(false, IonReaderContinuableTopLevelBinaryTest::annotationsIterator, inputBytes);
+    }
+
+    /**
+     * Checks that the reader correctly reads as annotations symbol 0 and inline empty text, which require special
+     * FlexSyms.
+     */
+    private void readAnnotationsWithSpecialFlexSyms_1_1(boolean constructFromBytes, String inputBytes) throws Exception {
+        reader = readerForIon11(hexStringToByteArray(inputBytes), constructFromBytes);
+        SymbolToken emptyText = new SymbolTokenImpl("", -1);
+        SymbolToken symbolZero = new SymbolTokenImpl(null, 0);
+        assertSequence(
+            next(IonType.INT), annotations(""), intValue(0),
+            next(IonType.INT), annotationSymbols(symbolZero), intValue(0),
+            next(IonType.INT), annotationSymbols(emptyText, symbolZero), intValue(0),
+            next(IonType.INT), annotationSymbols(symbolZero, emptyText), intValue(0),
+            next(null)
+        );
+        closeAndCount();
+    }
+
+    @ParameterizedTest
+    @CsvSource({
+        "E7 01 80 50 E7 01 90 50 E8 01 80 01 90 50 E9 09 01 90 01 80 50",
+        "E7 02 00 80 50 E7 04 00 00 90 50 E8 08 00 00 00 80 02 00 90 50 E9 90 00 00 00 00 01 90 01 80 50"
```

Comment:
Can you please put each test case on a separate csv line so that they are run separately? It's easier to read and it's easier to debug a failure because the different test cases get run in isolation from each other.

Reply:
See my comment on your related comment. Structuring them this way reduces the number of distinct test methods we need, which I like. I can reformat the CSV lines to be more readable.